charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -80,8 +82,8 @@ data:
       baseUrl: https://backstage-backstage-{{ .Values.rhdhOperator.subscription.namespace }}.{{ include "cluster.domain" . }}
     backend:
       auth:
-        keys:
-          - secret: {{ printf "${%s}" .Values.rhdhOperator.secretRef.backstage.backendSecret }}
```

Comment:
Do we still need to generate the value for `backendSecret` in the `setup.sh` script?

Reply:
we'll need some variation of it, I'll work on this is a separate commit.